api/matrix.ts

```diff
@@ -0,0 +1,74 @@
+import got from '../libs/got'
+import { millify } from '../libs/utils'
+import { fetchMembersCount as fetchGitterMembersCount } from './gitter'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+const BRAND_COLOR = 'black'
+
+export default createBadgenHandler({
+  title: 'Matrix',
+  examples: {
+    '/matrix/members/%23rust/matrix.org': 'members',
```

Comment:
Looks like this `%23` can be omitted

```suggestion
    '/matrix/members/rust/matrix.org': 'members',
```

Reply:
Yeah, I put it there intentionally so folks are aware it is an optional thing. Do we still want git rid of it?

Reply:
I think people would always prefer the simpler form

Reply:
Done :heavy_check_mark: